CHANGELOG.md

```diff
@@ -27,13 +29,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Secure worker connections are no longer supported for Editor builds. They are still supported for packaged builds.
 
 ### Features:
-- New default Control GameMode in the Example Project. Two Teams compete to control points on the map, with help from NPC guards.
+- The GDK now uses SpatialOS SDK version [`14.6.1`](https://documentation.improbable.io/sdks-and-data/docs/release-notes#section-14-6-1).
+- Added support for the new standard SpatialOS Runtime, version `0.4.1`.
+- Added support for the new compatibility mode SpatialOS Runtime, version [`14.5.4`](https://forums.improbable.io/t/spatialos-13-runtime-release-notes-14-5-4/7333).
+- Added a new dropdown setting in SpatialGDK Editor Settings that you can use to choose Runtime variant. There is currently Standard and Compatibility Mode. Standard is default, Compatibility Mode can be used if any networking issues arise when updating to the latest GDK version.
```

Comment:
I think we have previously debated (could also be within my head) to have an entire new section on important features, where things like this would go. (Idea probably came from Epic and their release notes).
From the list below, I think I would move the example project there, this runtime change, and maybe the new editor workflows. Not sure who is the ultimate authority on changelog changes.

Reply:
e.g: I think you're referring to the release notes (e.g: [0.9.0](https://github.com/spatialos/UnrealGDK/releases/tag/0.9.0)) which contain a summary of headline features, followed by:

> For the full details on what’s new in this release, see the 0.9.0 changelog.

So I think no action is required here and that will be taken care of later by product owner @ernopp in conjunction with marketing.

I'm happy to modify the CHANGELOG further is there's a spec for how to do this though?

Reply:
Ah, that could be enough

Reply:
yes I think the summary release notes (as we did for 0.9) handle this well
I will draft them and share around